src/luaharfbuzz/face.c

```diff
@@ -398,6 +398,31 @@ static int face_ot_color_has_png(lua_State *L) {
   return 1;
 }
 
+static int face_ot_color_has_svg(lua_State *L) {
+  Face *f = (Face *)luaL_checkudata(L, 1, "harfbuzz.Face");
+
+  lua_pushboolean(L, hb_ot_color_has_svg(*f));
+  return 1;
+}
+
+static int face_ot_color_glyph_get_svg(lua_State *L) {
+  Face *f = (Face *)luaL_checkudata(L, 1, "harfbuzz.Face");
+  hb_codepoint_t gid = (hb_codepoint_t) luaL_checkinteger(L, 2);
+  hb_blob_t* blob = hb_ot_color_glyph_reference_svg(*f, gid);
+
+  if (hb_blob_get_length(blob) != 0) {
+    Blob *b = (Blob *)lua_newuserdata(L, sizeof(*b));
+    luaL_getmetatable(L, "harfbuzz.Blob");
+    lua_setmetatable(L, -2);
+
+    *b = hb_ot_color_glyph_reference_svg(*f, gid);
```

Comment:
That is probably a thinko in my part in the PNG code, but you can save the extra call to `hb_ot_color_glyph_reference_svg()` and reuse the `blob` variable.
```suggestion
    *b = blob;
```

Reply:
@khaledhosny I fully agree, but I thought that if I change it here, then it also should be changed in the PNG version to be consistent and that wouldn't really fit into this PR.

Reply:
I’d change both in a separate commit.

Reply:
@khaledhosny Done